File: Android/app/src/main/java/com/example/appprojeto/models/Usuario.kt
package com.example.appprojeto.models

import com.google.gson.annotations.SerializedName

class Usuario {
    @SerializedName("codigo")
    var codigo: Int = 0

    @SerializedName("nome")
    var nome: String = ""

    @SerializedName("data")
    var nascimento: String? = ""

    @SerializedName("foto")
    var foto: String? = ""
}

File: Xamarin/AppProjeto/AppProjeto/AppProjeto/Models/Usuario.cs
using Newtonsoft.Json;

namespace AppProjeto.Models
{
    public class Usuario
    {
        [JsonProperty("codigo")]
        public int Codigo { get; set; }

        [JsonProperty("nome")]
        public string Nome { get; set; }

        [JsonProperty("data")]
        public string Data { get; set; }

        [JsonProperty("foto")]
        public string Foto { get; set; }
    }
}
